sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -181,6 +181,10 @@ private Session(Executor executor, SessionContext sessionContext) {
             this.sessionContext = sessionContext;
         }
 
+        public SessionContext sessionContext() {
```

Comment:
Seeing the usage further down I think we should remove the sessionContext property here. I don't like this indirection `ConnectionContext -> startupMessage with userName/defaultSchema -> createSession -> access sessionContext - retrieve userName/defaultSchema from sessionContext`